sources/packages/web/src/views/supporting-user/SupportingInformation.vue

```diff
@@ -91,6 +134,53 @@ export default {
     const studentsLastName = ref("");
     const studentsDateOfBirth = ref();
     const initialData = ref();
+    const formioUtils = useFormioUtils();
+    const isFirstPage = ref(true);
+    const isLastPage = ref(false);
+    const showNav = ref(false);
+    let formInstance: any;
```

Comment:
Please declare this as `FormIOForm` and add to the interface the missing properties/events as needed to satisfy the `formInstance.options` and `formInstance.on` and so on.
There are many areas of the code still using `any` but for development refactoring the idea is to get rid of them as much as possible. When `any` is required we try to use `unknown` instead 😉

Reply:
ok. Done. Thanks!

Reply:
The line 149 must also be declared with the proper type, as shown below.
```ts
const formLoaded = async (form: FormIOForm)
```

Reply:
ok. Done!